internal/service/api/handlers/create_identity.go

```diff
@@ -194,25 +193,26 @@ func CreateIdentity(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// timestamp is only 6 bytes long, if using some other salt, make sure that it is
+	// < 32 to be compatible with Poseidon hash function
 	salt := new(big.Int).SetUint64(uint64(time.Now().UTC().UnixMilli()))
-	documentHash, err := hashDocument(req.Data.DocumentSOD.SignedAttributes)
+	documentHash, err := poseidon.HashBytes(signedAttributes)
```

Comment:
Convert `signedAttributes` to `*big.Int` (if it is possible considering variable's size) and use `poseidon.Hash()` instead of `poseidon.HashBytes()`

Reply:
We will get `inputs values not inside Finite Field` as the value is more than 32 bytes long (its length is 74), so it is better to use HashBytes because it is designed to hash values of bigger size. Moreover, this hash encapsulated from other logic and we won't hash document in circuts.